### ISSUE TITLE
Add alt text to html code

### ISSUE THREAD

**Issue by dom-jackson** [dom-jackson/horiseon-accessibility#2]
html file does not include alternative text for the current content. Alt text is necessary as it describes the appearance or function of an image on a page. Alt text is read aloud by screen readers used by visually impaired users, displays in place of an image if it fails to load and is indexed by search engine bots to better understand the content of your page.

alt txt is to be added to all images and required content.

**Comment by dom-jackson** [comment]
Alt text added to the code. [d87c049] <img width="1000" alt="alt text added -  d87c049" src="https://user-images.githubusercontent.com/117177733/202983573-3c71ed48-16e9-47b3-bed0-19c70d59802f.png">
